docs/source/GETTING_STARTED.rst

```diff
@@ -22,11 +22,11 @@ Google Cloud Setup
 
 .. note:: The recent success of deep learning has been critically dependent on accelerated hardware like GPUs. Similarly, the strength of the DeepCell Kiosk is its ability to recruit and scale GPU nodes based on demand. Google does not include these GPU nodes by default as part of its free tier thus necessitating the upgrade. For more information, please refer to `Google's blog post on the subject <https://cloud.google.com/blog/products/gcp/gpus-service-kubernetes-engine-are-now-generally-available>`_
 
-4. You will also need to `apply <https://cloud.google.com/compute/quotas>`_ for a quota of at least 1 GPU using the ``GPUs (all regions)`` option. Please also request at least 16 *In-use IP addresses* for the *Compute Engine API* of your region (by default ``us-west1``). This may take some time, as Google will need to approve each these requests.
+4. Apply for a `quota of at least 1 GPU <https://cloud.google.com/compute/quotas#gpus>`_ and at least `16 *In-use IP addresses* <https://cloud.google.com/compute/quotas#ip_addresses>`_ for the *Compute Engine API* of your region (by default ``us-west1``). This may take some time, as Google will need to approve each of these requests.
 
 .. note:: Google offers a number of GPU types. The DeepCell Kiosk uses `nvidia-tesla-t4` GPUs for inference by default.
```

Comment:
I think it would be more clear if this 'note' was indented to the same level at item #4, so it's clear that's what its referring to. 

Also, GCP calls them NVIDIA T4 GPUs, without the tesla or the dashes; this makes it searchable from the quotas menu. It's still not totally obvious how one would add GPUs based on the current GCP links, but I understand we don't want to add information that may get updated as google changes how they do things.

Reply:
I've updated the name of the GPU, good call.

Which 'note' are you referring to?

Reply:
![image](https://user-images.githubusercontent.com/13770365/78407363-70cb6980-75b9-11ea-823d-0453c8360255.png)

Reply:
Just having it be part of the bullet rather than broken out

Reply:
optional though

Reply:
Ah it actually renders differently in RTD:

<img width="643" alt="Screen Shot 2020-04-03 at 4 32 00 PM" src="https://user-images.githubusercontent.com/7930703/78412624-e0952080-75c8-11ea-9c68-395182dbf6ca.png">